Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -535,16 +479,8 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner {
                             command: command,
                             output: String(decoding: stderrData, as: UTF8.self))
                     }
-                    // Otherwise we return a result based on its exit code. If
```

Comment:
This code is now hoisted up to PluginInvocation, which deals with the semantics, leaving the plugin script runner to only deal with the mechanics of communicating with the plugin.